src/Serializer/TaskNormalizer.php

```diff
@@ -98,11 +86,48 @@ public function normalize($object, $format = null, array $context = array())
             $delivery = $object->getDelivery();
 
             if (null !== $delivery) {
-                $data['packages'] = $this->_normalizeTaskPackages($delivery->getPackages());
-                $data['weight'] = $delivery->getWeight();
+                $deliveryId = $delivery->getId();
+
+                $qb =  $this->entityManager
+                ->getRepository(Task::class)
+                ->createQueryBuilder('t');
+
+                $query = $qb
+                    ->select('p.name AS name', 'p.name AS type', 'sum(tp.quantity) AS quantity')
+                    ->join('t.packages', 'tp', 'WITH', 'tp.task = t.id')
+                    ->join('tp.package', 'p', 'WITH', 'tp.package = p.id')
+                    ->join('t.delivery', 'd', 'WITH', 'd.id = :deliveryId')
+                    ->groupBy('p.name')
```

Comment:
should be grouped by `package.id` maybe ?

because we can have different packages with the same name ?